internal/stokerhttp/get.go

```diff
@@ -105,52 +123,25 @@ func (h *handler) getSteamapp(w http.ResponseWriter, r *http.Request) error {
 
 	parsedSteamappAppID, err := strconv.Atoi(steamappID)
 	if err != nil {
-		return newHTTPStatusCodeError(fmt.Errorf("parse Steamapp ID: %w", err), http.StatusBadRequest)
+		return newHTTPStatusCodeError(fmt.Errorf("parse steamapp ID: %w", err), http.StatusBadRequest)
 	}
 
-	row, err := h.Database.SelectBuildImageOpts(r.Context(), parsedSteamappAppID)
+	specRow, err := h.Database.SelectBuildImageOpts(r.Context(), parsedSteamappAppID)
 	if err != nil {
 		return fmt.Errorf("select build image options: %w", err)
 	}
 
-	metadata, err := getSteamappMetadata(r.Context(), row)
+	infoRow, err := h.Database.SelectSteamappInfo(r.Context(), parsedSteamappAppID)
```

Comment:
We could/should reduce this to one join query, but we'd have to settle on new model(s), so I didn't take it that far yet